### DISCUSSION TITLE
language of the traduction

### DISCUSSION THREAD

**Discussion by diabitbol** [crimx/ext-saladict#1150] [Ideas]
Hello I really like this project and I wanted to know if ext-saladict can traduce to another language than chinese. If you want I can speak english french and spanish.

**Comment by crimx** [comment]
Hi, what do you mean by "traduce to another language"?

**Reply by diabitbol** [reply]
You're right, sorry I meant "translate to another language than Chinese". For instance from English to French or from English to Spanish

**Reply by crimx** [reply]
There are no English to French nor English to Spanish dictionaries in Saladict at the moment. I may add them in the future but I can't promise anything now. There are many other features and bug-fixes that need to be done first.

**Reply by diabitbol** [reply]
that's ok
